test/onnx/test_operators.py

```diff
@@ -1013,6 +1021,45 @@ def forward(self, x, y):
 
         unregister_custom_op_symbolic('::embedding', _onnx_opset_version)
 
+    # This test is for shapeValueMap.
```

Comment:
This sentence is pretty redundant with the name of the test and the rest of the text is clear without it; I'd remove this sentence.